Atomic/trust.py

```diff
@@ -478,12 +479,13 @@ def get_gpg_id(self, keys):
         keylist = None
         for key in keys:
             if not os.path.exists(key):
-                # extend when we support parsing inline 'keyData' base64 encoded key
-                return ""
-            else:
-                cmd = ["gpg2", "--with-colons", key]
+                with tempfile.TemporaryFile() as tmpkey:
+                    tmpkey.write(b64decode(key))
+                key = tmpkey.name
+            cmd = ["gpg2", "--with-colons", key]
             try:
                 results = util.check_output(cmd).decode('utf-8')
+                tmpkey.close()
```

Comment:
@rhatdan I tightened the tempfile up a bit. I'm not sure what you mean by using /run. Prefix dir for tempfile?